### PR TITLE
removed deprecated cla info

### PR DESCRIPTION
**Pull request by Kawohl**
This will need some more dept investigation with the new release of ownCloud.org 
Just a hot fix for now.

### CODE REVIEW

File: page-agreement.php [hunk 1]
```diff
@@ -11,20 +11,15 @@
...
-called apps. We don't require a contributor agreement for the apps, only for the core, the desktop clients and android.<p>
-<h3>But I want to contribute to ownCloud and can&#39;t or don&#39;t want to sign a contributor agreement. What can I do?</h3>
+<p>No. 
 <p>We can accept your core contribution if you release your code under the MIT license. Of course we 
 recommend that you sign the agreement because AGPL/GPL protects the freedom of your work better than MIT. But the choice is yours.<p>
 <h3>How difficult is it to sign the agreement?</h3>
-Very easy. Take the PDF, sign it and send it by mail or scan to cla@owncloud.com</p>
+Very easy. When you submit your pull request you will prompted to sign the cla by clicking a button. We Use <a href="https://cla-assistant.io/" cla-assistant.io </a> for this
+</p>
 <p>or<br />
```
Comment (DeepDiver1975) [inline page-agreement.php]: there is no _or_

File: page-agreement.php [hunk 2]
```diff
@@ -11,20 +11,15 @@
...
-ownCloud GmbH<br />
-Leiziger Platz 21<br />
-90491 Nürnberg<br />
-Germany</p>
-<p>We will give you full commit rights as soon as we have received your agreement. If you also want to contribute
-to the ownCloud iOS app please also send the signed "iOS Addendum" document. See the <a href="/contribute/iOS-license-exception/">iOS license exception</a> for testing the ownCloud iOS app on Apple hardware.</p>
+
+<p>We will give you full commit rights as soon as we have received your agreement.
```
Comment (DeepDiver1975) [inline page-agreement.php]: Kill that line as well - we don't do this anymore for a long time. And there is also no need to ...

File: page-agreement.php [hunk 3]
```diff
@@ -11,20 +11,13 @@
...
-Very easy. Take the PDF, sign it and send it by mail or scan to cla@owncloud.com</p>
-<p>or<br />
-ownCloud GmbH<br />
-Leiziger Platz 21<br />
-90491 Nürnberg<br />
-Germany</p>
-<p>We will give you full commit rights as soon as we have received your agreement. If you also want to contribute
-to the ownCloud iOS app please also send the signed "iOS Addendum" document. See the <a href="/contribute/iOS-license-exception/">iOS license exception</a> for testing the ownCloud iOS app on Apple hardware.</p>
+Very easy. When you submit your pull request you will prompted to sign the cla by clicking a button. We Use <a href="https://cla-assistant.io/" cla-assistant.io </a> for this
+</p>
```
Comment (DeepDiver1975) [inline page-agreement.php]: this closing p tag is never opened ....

File: page-agreement.php [hunk 4]
```diff
@@ -11,20 +11,13 @@
...
+<p><br />
```
Comment (DeepDiver1975) [inline page-agreement.php]: open p but no closing ?

File: page-agreement.php [hunk 5]
```diff
@@ -11,20 +11,13 @@
 <p>The agreement allows the company to offer customers ownCloud under another license, in exchange for money.
 This thus helps pay the bills for ownCloud contributors.</p>
 <h3>Does this mean that everybody who wants to contribute to ownCloud has to sign this agreement?</h3>
-<p>No. Only the <a href="https://github.com/owncloud/core">ownCloud core repository</a> is covered by this contributor agreement. ownCloud is build out of a lot of small modules
-called apps. We don't require a contributor agreement for the apps, only for the core, the desktop clients and android.<p>
-<h3>But I want to contribute to ownCloud and can&#39;t or don&#39;t want to sign a contributor agreement. What can I do?</h3>
+<p>No.
```
Comment (lefherz) [inline page-agreement.php]: Not sure if I'm viewing the current version of this file, but what's up with these \<p\> tags?

File: page-agreement.php [hunk 6]
```diff
@@ -11,20 +11,13 @@
...
 <h3>How difficult is it to sign the agreement?</h3>
-Very easy. Take the PDF, sign it and send it by mail or scan to cla@owncloud.com</p>
-<p>or<br />
-ownCloud GmbH<br />
-Leiziger Platz 21<br />
-90491 Nürnberg<br />
-Germany</p>
-<p>We will give you full commit rights as soon as we have received your agreement. If you also want to contribute
-to the ownCloud iOS app please also send the signed "iOS Addendum" document. See the <a href="/contribute/iOS-license-exception/">iOS license exception</a> for testing the ownCloud iOS app on Apple hardware.</p>
+<p>Very easy. When you submit your pull request you will prompted to sign the cla by clicking a button. We Use <a href="https://cla-assistant.io/" cla-assistant.io </a> for this
```
Comment (lefherz) [inline page-agreement.php]: there is a \> missing in the \<a href\> tag and you should put a . at the end of the sentence, I guess.

File: page-agreement.php (hunk 5 above)
Comment (lefherz) [inline page-agreement.php]: In the two lines below it neither looks right. including this line, there are 3 opening \<p\> tags, and none of them are closed. are they working dynamically? I'm not thats sure about my html knowledge, maybe it works this way.

**Comment by Kawohl** [comment]
Even the content legacy is coded in a crappy way... .

**Comment by Kawohl** [comment]
@lefherz anything more?

**Comment by lefherz** [comment]
Nope, I'm fine with it now. @DeepDiver1975 too?

**Comment by DeepDiver1975** [comment]
@Kawohl @lefherz merge if you like ....

may I ask what editor you guys are using? html syntax errors are easier to spot with a proper editor .... just saying

**Comment by lefherz** [comment]
I myself? vim. love it. this stuff I spotted in the github UI though, my eyes aren't that bad yet, so...

**Comment by DeepDiver1975** [comment]
> my eyes aren't that bad yet, so...

well - they didn't spot the syntax error ;-)

**Pull request merged**